File: mgyoutube-multitech/api-java/src/main/java/com/djpedesen/mgyoutube/api_java/modules/UserNotFoundException.java
package com.djpedesen.mgyoutube.api_java.modules;

public class UserNotFoundException extends Exception {

	private static final long serialVersionUID = 7319712820184105763L;

	public String username = null;
	public long userId = Long.MIN_VALUE;

	public UserNotFoundException() {
	}

	public UserNotFoundException(final String username) {
		this.username = username;
	}

	public UserNotFoundException(final long userId) {
		this.userId = userId;
	}

}

File: mgyoutube-multitech/api-dotnet/modules/UserNotFoundException.cs
namespace api_dotnet.modules
{
    public class UserNotFoundException : System.Exception
    {
        public readonly string username;

        public UserNotFoundException(string username) : base()
        {
            this.username = username;
        }
        public UserNotFoundException(string username, string message) : base(message)
        {
            this.username = username;
        }
    }
}